nornir_nautobot/plugins/tasks/dispatcher/default.py

```diff
@@ -276,6 +276,94 @@ def _save_file(logger, backup_file: str, _running_config: str) -> None:
         with open(backup_file, "w", encoding="utf8") as filehandler:
             filehandler.write(_running_config)
 
+    @staticmethod
+    def provision_config(
```

Comment:
I know this exists already, but I think we should deprecate `provision_config` (remove in next major version) and add `replace_config`. We would keep both for now by just adding 

```
provision_config = replace_config
```